lib/observer/test/etop_SUITE.erl

```diff
@@ -116,9 +195,267 @@ text_tracing_off(Config) when is_list(Config) ->
     etop:config(sort,runtime), % this should not crash, but has no effect
     timer:sleep(3000),
     ok.
-text_tracing_off(cleanup,_Config) ->
-    etop:stop(),
-    {ok,Host} = inet:gethostname(),
-    Node = list_to_atom("node2@"++Host),
-    test_server:stop_node(Node).
 
+text_in_terminal(suite) ->
+    [];
+text_in_terminal(doc) ->
+    ["Start etop with shell, and verify that previous content is being overwritten"];
+text_in_terminal(Config) when is_list(Config) ->
+    Node = proplists:get_value(node, Config),
+    Term = proplists:get_value(term, Config),
+
+    EtopStartCmd = "spawn_link(etop, start, [[{node,"++atom_to_list(Node)++"},{output,text},{interval,1}]]).",
+    shell_test_lib:send_tty(Term, EtopStartCmd),
+    shell_test_lib:send_tty(Term, "Enter"),
+
+    timer:sleep(500),
```

Comment:
I've found that relying on sleeps for testing the shell is very error prone. It is usually better to use `shell_test_lib:check_content/2` to check that what is on the screen is correct. Not sure that the check_content algorithm works for something like etop, but you most likely will get issues if you use sleep.

Reply:
I think this is fixed as well, and there should be no random crashes in etop tests